docs/geedocs/5.3.1/answer/7160008.html

```diff
@@ -73,13 +71,69 @@ <h5>
             <th class="narrow">Description</th>
             <th>Resolution</th>
           </tr>
+          <tr>
+            <td>1236</td>
+            <td>Polygons and Polygon constructors should handle a KML string.</td>
+            <td>
+              Constructors now allows reading KML as string stream object in addition to reading it using file name.
+            </td>
+          </tr>          
+          <tr>
+            <td>1255</td>
+            <td>Version errors with GEE patch releases</td>
+            <td>
+              Limited internal version string to 3 parts to fix patch releases.
+            </td>
+          </tr>
+          <tr>
+            <td>1277</td>
```

Comment:
Same comment as above. Either remove or change description to indicate how this helps end users.